### ISSUE TITLE
Application does not start when Node 14 is installed

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#14661]
### Description of the bug

With node version `v14.19.0` in use you see the following when trying to start an application
```
[INFO] --- vaadin-maven-plugin:23.2.2:prepare-frontend (default) @ nodetest ---
[INFO] The globally installed Node.js version 14.19.0 is older than the required minimum version 16.14.0. Using Node.js from /Users/artur/.vaadin.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.871 s
[INFO] Finished at: 2022-09-27T16:12:29+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.vaadin:vaadin-maven-plugin:23.2.2:prepare-frontend (default) on project nodetest: Could not execute prepare-frontend goal.: 
[ERROR] 
[ERROR] ======================================================================================================
[ERROR] Your installed 'npm' version (6.14.16) is too old. Supported versions are 8.3+
[ERROR] Please install a new one either:
[ERROR]   - by following the https://nodejs.org/en/download/ guide to install it globally
[ERROR]   - or by running the frontend-maven-plugin goal to install it in this project:
[ERROR]   $ mvn com.github.eirslett:frontend-maven-plugin:1.10.0:install-node-and-npm -DnodeVersion="v16.17.0" 
[ERROR] 
[ERROR] You can disable the version check using -Dvaadin.ignoreVersionChecks=true
[ERROR] ======================================================================================================
```

### Expected behavior

1. The npm version installed in ~/.vaadin/node is used
2. There is no mention of needing a global node installation or `frontend-maven-plugin`

### Minimal reproducible example

Start the default app from start.vaadin.com

### Versions

- Vaadin / Flow version: 23.2.2

**Comment by Artur-** [comment]
Might be related to #14636 as the node version installed in ~/.vaadin/node was 18.9.1

Removing it installs v16.17.0 instead and the app starts

**Comment by caalador** [comment]
Does this issue persist or was it due to a broken node 18 installation?

**Comment by Artur-** [comment]
If was fixed with #14636
